src/corg/developer-environment-setup/installing-git.md

```diff
@@ -295,9 +295,10 @@ Please don't put accents and such in your username (as well in the commit messag
 
 ### SSH Key
 
-On the GitHub site, click **Account Settings**&nbsp;> Click **SSH Public Keys**&nbsp;> Click **Add another public key** and add your public key there.
+At Nuxeo, we use SSH over HTTPS for data transfer with GitHub. Therefore, you need to configure your SSH key.
```

Comment:
Yes, you're right. We just use SSH, so that's why I said _"we use SSH **over** HTTPS"_. Maybe I can rework the expression into either:

- "we prefer SSH over HTTPS for data transfer with GitHub."
- "we use SSH instead of HTTPS for data transfer with GitHub."